### PR TITLE
[#265][BUGFIX] Correction de la sauvegarde des réponses front (US-355)

### PR DESCRIPTION
**Pull request by bdavidxyz**

### CODE REVIEW

**Comment by jbuget** [comment]
I've deployed this PR to http://debug-circleci.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://debug-circleci.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://debug-circleci.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://debug-circleci.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://debug-circleci.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://debug-circleci.pix.beta.gouv.fr. Please check it out

File: api/tests/acceptance/application/answer-controller-find_test.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/* global describe, before, beforeEach, after, afterEach, knex, it, expect */
+const server = require('../../../server');
+
+server.register(require('inject-then'));
+
+describe('Acceptance | API | Answers', function () {
+
+  before(function (done) {
+    knex.migrate.latest().then(() => {
```
Comment (MBrandone) [inline api/tests/acceptance/application/answer-controller-find_test.js]: verifier que la migration n'est pas déjà lancée dans le package.json

**Comment by jbuget** [comment]
I've deployed this PR to http://debug-circleci.pix.beta.gouv.fr. Please check it out

**Pull request merged**